fabric/README.md

```diff
@@ -46,7 +46,12 @@ Place your client certificate and your SPID in the ``ias`` folder.
     echo 'YOURSPID' | xxd -r -p > /path-to/fabric/sgxconfig/ias/spid.txt
 
 We currently make use of `unlinkable signatures` for the attestation, thus, when registering with the IAS please choose
-unlinkable signatures.
+unlinkable signatures.  In the case you prefer linkable attestation,
+e.g., because you already have linkable IAS EPID credentials, change
+in line 217 of [../ecc_enclave/sgxcclib/sgxcclib.c](../ecc_enclave/sgxcclib/sgxcclib.c)
+the constant `SGX_UNLINKABLE_SIGNATURE` to `SGX_LINKABLE_SIGNATURE`,
+recompile ercc, configure your IAS settings as above with your
```

Comment:
it should be ecc_enclave/ecc instead or ercc

Reply:
@mbrandenburger  Fixed to point to ecc and ecc_enclave an added corresponding links.

PS: i noticed you added an explicit merge and also prior PRs have mostly an additional empty merge commit. In PDO we try to minimize commits by rebasing and potentially squashing commits (e.g., i did incorporate the fix now in the original commit). We might at some point discuss our strategy to commits (e.g., best-practices) to get a consistent git log?